commands/hubdb/create.ts

```diff
@@ -51,20 +47,18 @@ exports.handler = async options => {
   } catch (e) {
     logger.error(
       i18n(`${i18nKey}.errors.create`, {
-        src,
+        providedFilePath,
       })
     );
     logError(e);
   }
 };
 
 exports.builder = yargs => {
-  addAccountOptions(yargs);
```

Comment:
Rather than removing these utils from each command file, you can actually keep them here and remove them from the top level definition [here](https://github.com/HubSpot/hubspot-cli/blob/main/commands/hubdb.ts#L15).

The benefit of removing them there is that it prevents those flags from showing up when you run `hs hubdb --help` (where they're useless anyways and just clutter up the console output)

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1267/commits/db6a7e1dbb0beb080e3ef4bcdd26db551c69d7db